GoogleForCreators/web-stories-wp
# Document Panel: Improve Calendar accessibility

## Summary
- Adds aria labels to prev-next buttons
- Close the calendar when using "Esc"
- Focus when opening the popup
- state aria-pressed information for AM/PM toggle
- use the roving tab index for Calendar days.


## Relevant Technical Choices



## To-do

- [ ] Karma



## User-facing changes



## Testing Instructions
1. Open the calendar
2. Verify that the first input got focused
3. Tab into the dates
4. Use arrow keys to move the focus
5. Click "Esc" to close the calendar
6. Verify that the aria labels exist for forward/back arrows.


---



Fixes #4431
